modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceDBPartitionTest.java

```diff
@@ -424,11 +460,13 @@ public void testCopyDBPartitionCompany() throws Exception {
 			safeCloseable.close();
 		}
 		finally {
-			companyLocalService.deleteCompany(company.getCompanyId());
-
 			if (copiedCompany != null) {
 				companyLocalService.deleteCompany(copiedCompany.getCompanyId());
 			}
+
+			if (configuration != null) {
```

Comment:
why this is needed now?

Reply:
Because before it was part of the added company that was removed at the end. With the latest changes it gets added as part of the partition default company that doesn't get removed.